sourcegraph/about
# Codecov Case Study/headshots

Closes #5635. Adds team headshots to Codecov Case Study & adjusts `<AuthorBio />` to have a multi-author design.

### Changelog
""

### Test
1. Ensure prettier has standardized the proposed changes.
2. Nav to `/case-studies/codecov-uses-sourcegraph-to-resolve-incidents-faster` and ensure headshots meet design
3. Ensure no breaking changes to the Nutanix case study `<AuthorBio />` section
